spec.md

```diff
@@ -398,6 +398,34 @@ Example error response (invalid `id` provided):
 
 ### Resource Data Model
 
+A resource is a named (addressable) blob stored in a Space, with metadata.
+The data model is derived from [W3C FileAPI: File 
+Interface](https://w3c.github.io/FileAPI/#file-section), but with the addition
+of a few crucial properties.
+
+In similar storage systems, a resource is called "File", "Object", "Document",
+"Row", "Graph", and so on.
+
+Resource properties:
+
+* `id`
+* `contents` (Blob)
+* `name` - optional
+* (tbd) a link to a **system-meta** auxiliary object (determined by the local
+  system)
+* (tbd) one or more links to **custom-meta** auxiliary objects (unlike
+  system-meta, these are modifiable by the object's controller)
```

Comment:
IMHO 'the object's controller' is ambiguous. Do you mean the auxiliary objects' controllers? or the resource's controller?

Also I would argue that all of them are modifiable by their controller, it's just that the system is the controller of some of them.

Reply:
Changed it to 'resource controller'